store/search/src/main/scala/org/apache/spark/rpc/Master.scala

```diff
@@ -81,7 +81,7 @@ class Master(sparkConf: SparkConf) {
           do {
             try {
               LOG.info(s"starting registry-service on $hostAddress:$port")
-              val config = RpcEnvConfig(
+              val config = RpcUtil.getRpcEnvConfig(
```

Comment:
RPC is refactored in #2372, this should be rebased after #2372 is merged

Reply:
After analyzing the #2372 these changes are not required,so reverted